titration/utils/ui_state/user_value/user_value.py

```diff
@@ -27,9 +27,14 @@ def __init__(self, titrator, previous_state, message):
             message (string): the message is used to display what setting you are entering
         """
         super().__init__(titrator, previous_state)
-        self.message = message
         self.string = ""
```

Comment:
Are all values strings? I think this could have a better name.

Reply:
Change the name to value